src/pages/api/auth/form.js

```diff
@@ -0,0 +1,19 @@
+export default function handler(req, res) {
```

Comment:
ALL files should be `.ts` (Non Component) or `.tsx` (JSX component) - No JS files!